Since setup is already defining _POSIX_C_SOURCE, also define
_DARWIN_C_SOURCE to avoid this error:


/usr/include/signal.h:69:42: error: use of undeclared identifier 'NSIG'
extern __const char *__const sys_signame[NSIG];
                                         ^
/usr/include/signal.h:70:42: error: use of undeclared identifier 'NSIG'
extern __const char *__const sys_siglist[NSIG];
                                         ^

https://github.com/mypaint/mypaint/issues/1086
https://github.com/mypaint/mypaint/commit/c1ff5f8ec1b428a663f578405d08180a5df31379
--- setup.py.orig	2020-05-29 11:40:02.000000000 -0500
+++ setup.py	2020-06-02 21:15:33.000000000 -0500
@@ -874,7 +874,7 @@
     extra_link_args = []
 
     if sys.platform == "darwin":
-        pass
+        extra_compile_args.append('-D_DARWIN_C_SOURCE')
     elif sys.platform == "win32":
         pass
     elif sys.platform == "msys":
